dist/routable-component.js

```diff
@@ -310,9 +310,9 @@ var routableComponent =
 	     */
 	
 	  }, {
-	    key: 'controller_classes',
+	    key: 'controllerClasses',
 	    get: function get() {
-	      return this._controller_classes;
+	      return this._controllerClasses;
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->